pal/src/cruntime/finite.cpp

```diff
@@ -106,8 +98,8 @@ _isnan(
 
 See MSDN doc
 --*/
-double
-__cdecl
+double 
+__cdecl 
```

Comment:
Yes, I simply reverted to previous version. It contains spaces but would be identical and easier to merge back to master.